frontend/src/docs/DocsWrapper.tsx

```diff
@@ -89,6 +93,19 @@ export default function DocsWrapper({ children }: DocsWrapperProps): ReactElemen
     push('/docs')
   }
 
+  function returnNewIndex(currentIndex: number, direction: DirectionalNavigation) {
+    let increment = 0 + direction
+    let index: undefined | number = undefined
+    while (index === undefined && currentIndex + increment >= 0 && currentIndex + increment < flatDirectory.length) {
+      if (!('header' in flatDirectory[currentIndex + increment])) {
+        index = currentIndex + increment
+      }
+      increment += direction
+    }
+
+    return index === undefined ? 0 : index
```

Comment:
Can be shortened to `index ? 0 : index`

Reply:
Tried that already but because index can be undefined - it throws an error when calling `changePage`. Very annoying.